bombastic/api/src/lib.rs

```diff
@@ -33,7 +36,10 @@ pub struct Run {
     pub index: IndexConfig,
 
     #[command(flatten)]
-    pub storage: StorageConfig,
+    pub sbom_storage: StorageConfig,
+
+    #[arg(skip)] // TODO: not this!!!
+    pub vex_storage: StorageConfig,
```

Comment:
@ctron I could use some help here. I needed to pull in a storage config for the VEX docs in addition to the SBOM's, but I can't `command(flatten)` both of them without name collisions. I was hoping clap offered some type of prefix feature I could use to distinguish them, but I couldn't find it. Any idea?

Reply:
No, I ran into the same with various times. Clap is a command line parser, not a config engine … I miss what we had in Drogue (the `config` crate?). I am not so sure handling it all with `clap` is the best approach.